src/util/schemas/GuildUpdateSchema.ts

```diff
@@ -1,26 +1,24 @@
 /*
 	Spacebar: A FOSS re-implementation and extension of the Discord.com backend.
 	Copyright (C) 2023 Spacebar and Spacebar Contributors
-	
+
 	This program is free software: you can redistribute it and/or modify
 	it under the terms of the GNU Affero General Public License as published
 	by the Free Software Foundation, either version 3 of the License, or
 	(at your option) any later version.
-	
+
 	This program is distributed in the hope that it will be useful,
 	but WITHOUT ANY WARRANTY; without even the implied warranty of
 	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 	GNU Affero General Public License for more details.
-	
+
 	You should have received a copy of the GNU Affero General Public License
 	along with this program.  If not, see <https://www.gnu.org/licenses/>.
 */
 
 import { GuildCreateSchema } from "@spacebar/util";
 
-export interface GuildUpdateSchema
-	extends Omit<GuildCreateSchema, "channels" | "name"> {
-	name?: string;
+export interface GuildUpdateSchema extends Omit<GuildCreateSchema, "channels"> {
```

Comment:
Simplified this, the `name` property has the same type for both so why not reuse it.